Src/BootCamp.Chapter/CalculatorSimulator.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class CalculatorSimulator
+    {
+        private ILogger _logger;
```

Comment:
Codacy found an issue: [Make '_logger' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5742055)